### PR TITLE
RegisterPage(focus, completeMessage)

### PR DESCRIPTION
**Pull request by chepchep2**
RegisterPage에서 
focus를 이동시키는 것과 버튼의 클릭 효과를 없애는 것, 
비밀번호 확인 textField에서 mainPage가기 전 비밀번호 확인 완료! 메세지를 보이는 것 까지 작업하였습니다.
- 아직 다음 textField로 이동하면서 키보드가 자동으로 활성화되는 것은 구현하지 못했습니다. 비밀번호 찾기 페이지 작업이 덜 되어 그 작업부터 하고 다시 작업 하려고 합니다.
- 구현 영상입니다. https://imgur.com/Vsu6vQp

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#64** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/64?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#65](https://github.com/team-stoodi/MyBaseballRecordClient/pull/65) <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/65?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#66](https://github.com/team-stoodi/MyBaseballRecordClient/pull/66) <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/66?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#63** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/63?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#62** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/62?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#61** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/61?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#60** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/60?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#59** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/59?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#58** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/58?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#57** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/57?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#56** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/56?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#55** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/55?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#54** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/54?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#53** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/53?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#52** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/52?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#51** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/51?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#50** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/50?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#49** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/49?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#48** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/48?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#47** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/47?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#46** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/46?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#45** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/45?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#44** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/44?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#43** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/43?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#42** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/42?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#41** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/41?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#40** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/40?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#39** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/39?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#35** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/35?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#34** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/34?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/page/register_page.dart [hunk 1]
```diff
@@ -201,29 +207,37 @@ class _RegisterPageState extends State<RegisterPage> {
                       controller: confirmPasswordController,
                       onClearPressed: () => confirmPasswordController.clear(),
                       keyboardType: TextInputType.text,
-                      onChanged: (String value) {},
+                      onChanged: (String value) {
```
Comment (flowkater) [inline lib/page/register_page.dart:210]: 함수로 분리하면 좋을 것 같습니다.
```dart
setState(() {
  showFinalPassword = value == passwordController.text;
});
```

이렇게 리팩토링 하면 더 좋지 않을까요?

**Review by flowkater — CHANGES_REQUESTED** [review]
로직이 너무 덕지덕지 복잡해지는 것 같네요.
각 if 문마다 왜 그렇게 동작해야하는지 상세하게 설명해주시면 감사하겠습니다.
조금 더 코드 정리가 필요해보이네요.

File: lib/page/register_page.dart (hunk 1 above)
Comment (chepchep2) [reply lib/page/register_page.dart:210]: 네 알겠습니다.

**Comment by chepchep2** [comment]
### Merge activity

* **Apr 7, 10:24 AM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/35).
* **Apr 7, 10:26 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/35) rebased this pull request as part of a merge.
* **Apr 7, 10:27 AM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/35).

**Pull request merged**